src/main/resources/schemas/exchange/organizationDirectory.xsd

```diff
@@ -0,0 +1,78 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<xsd:schema xmlns:xsd="http://www.w3.org/2001/XMLSchema"
+           xmlns:tns="urn:gov:hhs:fha:nhinc:exchange:directory"
+           targetNamespace="urn:gov:hhs:fha:nhinc:exchange:directory"
+           elementFormDefault="qualified">
+
+  <xsd:complexType name="organization">
+    <xsd:sequence>
+      <xsd:element name="name" type="xsd:string"/>
+      <xsd:element name="hcid" type="xsd:string"/>
+      <!-- Organization status for example, active -->
+      <xsd:element name="status" type="xsd:string"/>
+      <xsd:element name="contact" type="tns:contactType" minOccurs="0" maxOccurs="unbounded"/>
+      <xsd:element name="address" type="tns:addressType" minOccurs="0" maxOccurs="unbounded"/>
+      <xsd:element name="partOf" type="tns:partOfType" minOccurs="0"/>
+      <xsd:element name="endpoint" type="tns:endpointType" minOccurs="0" maxOccurs="unbounded"/>
```

Comment:
Again here, I'd have an endpointList element and then each element to separate it out.